pdd/horovod_trainer.py

```diff
@@ -0,0 +1,173 @@
+from .metrics import knn_acc
```

Comment:
horovod trainer?

Reply:
what is the difference between this file and trainer.py? We can add an horovod option and enable it inside the TripletTrainer class